hyo2/openbst/lib/processing/process_management/process_manager.py

```diff
@@ -0,0 +1,250 @@
+import logging
+
+from enum import Enum
+from netCDF4 import Dataset
+
+from hyo2.openbst.lib.processing.parameters import Parameters
+from hyo2.openbst.lib.processing.process_methods.dicts import ProcessMethods, process_requirements
+
+logger = logging.getLogger(__name__)
+
+
+class ProcessStageStatus(Enum):
+    # Already in processing chain
+    PRIORPROCESS = 0
+    REPEATEPROCESS = 1
+
+    # Not in processing chain
+    FIRSTPROCESS = 2
+    NEWPROCESS = 3
+    MODIFIEDPROCESS = 4
+
+
+class ProcessManager:
```

Comment:
What is the role of this `ProcessManager` class? What is its relationship with the `Process` class?